source/_integrations/lutron.markdown

```diff
@@ -22,6 +23,8 @@ Presently, there's only support for communicating with the [RadioRA 2](http://ww
 
 When configured, the `lutron` integration will automatically discover the rooms and their associated switches/dimmers as configured by the RadioRA 2 software from Lutron. Each room will be treated as a separate group.
 
+This component supports importing Areas defined by the Lutron RadioRA2 system into Home Assistant.  You can enable this feature in the integration UI.
```

Comment:
Please describe to the user how to do that. e.g., using click paths. Numerous examples can be found in our existing documentation.

Reply:
Done